ci/pipeline.yaml.erb

```diff
@@ -279,14 +278,36 @@ jobs:
           image: image/image.tar
           additional_tags: kubernikus.git/.git/ref
 
+  - name: replicate-images
+    plan:
+    - get: kubernikus.git
+      trigger: true
+      passed: [build]
+    - task: replicate
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: keppel.eu-de-1.cloud.sap/ccloud-dockerhub-mirror/sapcc/unified-kubernetes-toolbox
+            tag: "20220714153155"
+        inputs:
+        - name: kubernikus.git
+        run:
+          path: /bin/sh
+          args: 
+          - -cx
+          - |
+            yq e '.imagesForVersion.*.*|.repository + ":" + .tag' kubernikus.git/charts/images.yaml | grep -v changeme | sed -E 's/\$REGION|global/eu-de-1/g' | xargs -n1 -P8 keppel validate
```

Comment:
Is that lone `e` argument in the beginning a typo?

Reply:
No, that’s `yq` „golang edition“. It became optional in later versions…